ayekan/prometheus/prometheus.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#! /usr/bin/env bash
```

Comment:
The space after the `#!` is much less common than no space.

Reply:
fixed it in the last commit